db/migrate/20220810084526_create_post.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true, index: true
+      t.string :title
+      t.text :text
+      t.integer :commentsCounter
+      t.integer :likesCounter
```

Comment:
- [ ] please it would be better to follow the convention of creating columns in the database they should be in snake_case like that: comment_counter ...etc please try to do that